metricbeat/mb/testing/data/data_test.go

```diff
@@ -225,13 +267,43 @@ func checkDocumented(t *testing.T, data []common.MapStr) {
 	}
 }
 
+// omitDocumentedField returns true if 'field' is exactly like 'omitField' or if 'field' equals the prefix of 'omitField'
+// if the latter contains a dot.wildcard ".*". For example:
```

Comment:
Any chance you could add a table test for what you documented here? This is to make sure we don't break it in the future.

Reply:
Table test added